about.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
```

Comment:
- [ ] Just as in your index.html page, let's make sure that images are not flowing out of the page on the about section.

Reference:
![image](https://user-images.githubusercontent.com/79114233/142645276-bdd45962-1875-460a-b4d9-ee68082440b0.png)

Reply:
- [ ] Let's try to be more consistent with the design provided by not displaying the lists of partners on the `about` section and changing the footer to work with a dark background.

Yours:
![image](https://user-images.githubusercontent.com/79114233/142645676-f76d03d0-aa03-483f-8e29-2c89ad00156e.png)


Template:
![image](https://user-images.githubusercontent.com/79114233/142645643-5d98f68b-a4bd-41e6-a172-3cb08559ecb6.png)

Reply:
- [ ] Let's link the logo in your navbar to the home page. This ensures that the user can easily navigate through every place of your website, resulting in better User Experience.

- [ ] Let's also include a link to the home page on the mobile menu, so users in mobile can navigate just as easily.

Reply:
- [ ] Please, make sure that your mobile menu is working on both the index and the about section.

Reply:
- [ ] Lastly, but not least. Please include in your README file credits to the original design idea, as required by the [Creative Commons License.](https://creativecommons.org/licenses/)